BuyPolish/Pola/UI/ProductSearch/ScanCode/BPScanCodeView.m

```diff
@@ -126,9 +123,21 @@ - (void)setInfoTextVisible:(BOOL)visible {
     }];
 }
 
-- (void)setMenuButtonVisible:(BOOL)visible animation:(BOOL)animation {
+- (void)configureInfoLabelForMode:(BPScanCodeViewLabelMode)mode {
+    if (mode == BPScanCodeViewLabelModeScan) {
+        self.infoTextLabel.text = NSLocalizedString(@"Scan barcode", nil);
+    } else {
+        self.infoTextLabel.text = NSLocalizedString(@"Type 13 digits", nil);
+        [self setInfoTextVisible: YES];
```

Comment:
this method should not be called here, if you need to show it, call 2 methods: `configureInfoLabelForMode:` and `setInfoTextVisible:`